### DISCUSSION TITLE
FaceID, problem setting it up

### DISCUSSION THREAD

**Discussion by landrasss** [vladmandic/sdnext#2707] [Q&A]
I have a problem getting FaceID to work, I have already installed the requirements: `pip install insightface ip_adapter`, but it still does not appear in the script section. In the console I get the following error:

`Extension: script='scripts\faceid.py' 14:15:45-658335 ERROR`
`FaceID: No module named 'ip_adapter'`

That is. Thanks in advance.

**Comment by vladmandic** [comment]
this has been addressed in dev branch and will be part of the next release (faceid is fully automated, no need for manual setup of anything)

**Comment by landrasss** [comment]
Ok, I'll wait for the release. Thank you.

**Reply by vladmandic** [reply]
just fyi, release is pretty much ready but this time waiting for team to assemble some docs as well - there are sooo many new features, its becoming pointless to do releases if people don't know how to use it.

**Comment by cgidesign-de** [comment]
I installed the release from 2024-01-30 today. It is working but I get the same issue.
Is there a way to fix this issue manually before a new release?
And, dumb question, what is "faceid" supposed to do?

2024-02-07 21:32:14,588 | sd | DEBUG | webui | Load extensions
2024-02-07 21:32:14,759 | sd | ERROR | faceid | FaceID: No module named 'ip_adapter'
2024-02-07 21:32:14,761 | sd | INFO | script_loading | Extension: script='scripts\faceid.py' [2;36m21:32:14-759088[0m[2;36m [0m[1;31mERROR   [0m FaceID: No module named [32m'ip_adapter'[0m

**Reply by vladmandic** [reply]
all it needs is going to be auto-installed on first use.
what is it? a way to provide photo of a face and instantly use it to guide generation to look that way.

**Comment by cgidesign-de** [comment]
Thanks for the quick reply.
